crates/ide/src/goto_definition.rs

```diff
@@ -73,10 +73,15 @@ pub(crate) fn goto_definition(
         .into_iter()
         .filter_map(|token| {
             let parent = token.parent()?;
+
             if let Some(tt) = ast::TokenTree::cast(parent.clone()) {
                 if let Some(x) = try_lookup_include_path(sema, tt, token.clone(), file_id) {
                     return Some(vec![x]);
                 }
+
+                if let Some(x) = try_lookup_macro_def_in_macro_use(sema, token.clone()) {
```

Comment:
Fine to do now, but ideally this would be done the same way we handle derives, by giving `macro_use` a dummy expansion we can map into. https://github.com/rust-lang/rust-analyzer/issues/11929